tests/test_client.py

```diff
@@ -112,6 +113,21 @@ def test_cmd_timeout(self):
                                   f"({type(details)}) was raised "
                                   f"using command {cmd} ({self.cmds})")
 
+    @unittest.skipUnless(os.environ.get('AEXPECT_TIME_SENSITIVE'),
+                         "AEXPECT_TIME_SENSITIVE env variable not set")
+    def test_cmd_output_with_inner_timeout(self):
+        """
+        cmd_output_safe uses 0.5s inner timeout, make sure all lines are
```

Comment:
Perhaps it is better to just write "Make sure all lines are present in the output." in the main docstring, adding the `cmd_output_safe` subsentence below as a detail and complete separate sentence.

Reply:
Well this is cmd_output_safe-specific test, the read_nonblocking is not usually executed in a loop.

Reply:
Could that for instance rather be mentioned in `test_cmd_output_safe_with_inner_timeout`?